docs/monitoring-usage.rst

```diff
@@ -278,3 +278,91 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Metrics
+---------------------------
+
+The Stackdriver Monitoring API can be used to write data points to custom metrics. Please refer to
+the documentation on `Custom Metrics`_ for more information.
+
+To write a data point to a custom metric, you must provide an instances of
+:class:`~gcloud.monitoring.metric.Metric` created with a metric type as well as values for
+the labels associated with that metric type. The metric type can either
+be created using the :class:`~gcloud.monitoring.metric.MetricDescriptor` class or
+auto-created.  For more information on creating metric descriptors, see the `Metric Descriptors`_
+section. For information on metric type auto-creation, see `Auto-creation of custom metrics`_.
+
```

Comment:
(Responding here so I could point to the latest version.)

I see. I didn't see a response to it, so thought you might have missed it. The confusion regarding metric descriptor objects in the client library vs. metric descriptors as created through the API still remains. When talking about type / descriptor you can replace the last 3 sentences with something like:


You will either have created the metric descriptor earlier or be planning to take advantage of [auto-creation](https://cloud.google.com/monitoring/custom-metrics/creating-metrics#auto-creation).